File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int ans = 0;
    int k=0;
    int j=0;
    for(int i = 0; i<nums.length; i++){
        
        if(nums[i]>0){
            k++;
        }
        if(nums[i]<0){
            j++;
        }
    }
    ans = Math.max(k,j);
    return ans;
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maximumCount = function(nums) {
    let ans = 0;
    let k=0;
    let j=0;
    for(let i = 0; i<nums.length; i++){
        
        if(nums[i]>0){
            k++;
        }
        if(nums[i]<0){
            j++;
        }
    }
    ans = Math.max(k,j)
    return ans;
};